### PR TITLE
Update IAM policy to have the min requirement

### PR DESCRIPTION
**Pull request by boaz0**
Updating JSON to contain only the necessary permissions.

//cc @rvsia

### CODE REVIEW

File: packages/sources/src/addSourceWizard/hardcodedComponents/aws/arn.js [hunk 1]
```diff
@@ -84,34 +84,20 @@ export const IAMPolicyDescription = () => {
...
-                    Statement: [
-                        {
-                            Sid: 'VisualEditor0',
-                            Effect: 'Allow',
-                            Action: [
-                                's3:Get*',
-                                's3:List*'
-                            ],
-                            Resource: [
-                                `arn:aws:s3:::${s3Bucket}`,
```
Comment (rvsia) [inline packages/sources/src/addSourceWizard/hardcodedComponents/aws/arn.js:97]: s3Bucket si no longer used, so you can remove the formOptions from this component

File: packages/sources/src/addSourceWizard/hardcodedComponents/aws/arn.js [hunk 2]
```diff
@@ -84,34 +84,20 @@ export const IAMPolicyDescription = () => {
...
-                            Resource: '*'
-                        }
-                    ]
+                    "Version": "2012-10-17",
+                    "Statement": [{
+                        "Sid": "VisualEditor0",
+                        "Effect": "Allow",
+                        "Action": ["s3:Get*", "s3:List*"],
+                        "Resource": [
+                            "arn:aws:s3:::hccm-nise-data-01",
```
Comment (rvsia) [inline packages/sources/src/addSourceWizard/hardcodedComponents/aws/arn.js:93]: or it is? should not it be here?

Comment (rvsia) [reply packages/sources/src/addSourceWizard/hardcodedComponents/aws/arn.js:93]: same below

File: packages/sources/src/addSourceWizard/hardcodedComponents/aws/arn.js (hunk 1 above)
Comment (boaz0) [reply packages/sources/src/addSourceWizard/hardcodedComponents/aws/arn.js:97]: Right! I will fix it. Thanks

**Comment by boaz0** [comment]
Thanks @rvsia

**Pull request merged**